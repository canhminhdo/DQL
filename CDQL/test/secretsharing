#!/bin/sh

DIR=../test
cd ../examples && maude < secretsharing.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/secretsharing.out
diff $DIR/secretsharing.expected $DIR/secretsharing.out
rm $DIR/secretsharing.out