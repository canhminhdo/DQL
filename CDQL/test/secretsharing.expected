==========================================
reduce in SECRET-SHARING : initQState |= [H(1) ; CX(1,2) ; CX(1,3) ; (Alice ||
    Bob || Charlie)]P(3,qubitAt(initQState, 0)) .
result JudgmentSet: emptyJS
Bye.
